cmd/populator/cmd/register/ocf_manifests.go

```diff
@@ -89,30 +87,83 @@ func runDBPopulate(ctx context.Context, src string) (err error) {
 			err = multierror.Append(err, cErr)
 		}
 	}()
-
 	session := driver.NewSession(neo4j.SessionConfig{})
 	defer func() {
 		if sErr := session.Close(); sErr != nil {
 			err = multierror.Append(err, sErr)
 		}
 	}()
 
-	var gitHash []byte
-	if cfg.UpdateOnGitCommit {
-		log.Info("APP_UPDATE_ON_GIT_COMMIT set. Updating manifests only if git commit changed.")
-		gitHash, err = getGitHash(rootDir)
+	dataInDB, err := dbpopulator.IsDataInDB(session, log, commits)
```

Comment:
we should do it only if `cfg.UpdateOnGitCommit` is true 👍

Reply:
Yes, it was in my checklist for some reason I didn't review it correctly. Now it should be fine.